examples/quickstart/jupyter-notebooks/druidapi/druidapi/__init__.py

```diff
@@ -13,14 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .druid import DruidClient
+from druidapi.druid import DruidClient
```

Comment:
These changes are not needed. In Python, '.' means relative to the present module. This allows us to, say, install `druidapi` twice: `druidapi` and `druidApiOld` and have both work.

Reply:
Do we have a reason to prefer relative imports? It seems that the [recommended approach](https://peps.python.org/pep-0008/) is absolute imports when possible.
>Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):